src/client/Client.cc

```diff
@@ -11913,7 +11924,10 @@ int Client::_unlink(Inode *dir, const char *name, const UserPerm& perm)
   res = _lookup(dir, name, 0, &otherin, perm);
   if (res < 0)
     goto fail;
-  req->set_other_inode(otherin.get());
+
+  in = otherin.get();
+  req->set_other_inode(in);
+  in->break_deleg(false);
```

Comment:
as above, would the information conveyed by "false" be clearer if it were some kind of flag/constant?